packages/authentication/src/types.ts

```diff
@@ -22,7 +22,7 @@ export interface AuthenticationMetadata {
   /**
    * Name of the authentication strategy
    */
-  strategy: string;
+  strategy: string | string[];
```

Comment:
according to https://github.com/strongloop/loopback-next/pull/5735/files#r439852618, let's still keep the name as type `string`. One metadata object configures one strategy.